doc/howtos/rdtraining/B_acl_irrules.rst

```diff
@@ -4,4 +4,424 @@
+    intermediate groups, yet others may need something more complicated than a
+    monotonically increasing set of operations they can perform, ...
+
+For our purposes that baseline seems sufficient:
+
+* Real estate managers can configure the system (manage available types and
+  tags) as well as oversee every property in the pipeline.
+* Real estate agents can manage the properties under their care, or properties
+  which are not specifically under the care of an agent (properties without a
+  salesman set).
+
+In keeping with Odoo's data-driven nature, a group is no more than a record of
+the ``res.groups`` model. They are normally part of a module's "master data"
+and thus defined in one of the module's data files, refer to
+:ref:`the guidelines <reference/guidelines>` for the location these should go
+to.
+
+The most important fields of groups are:
+
+``name``
+    Serves as the user-readable identification for the group, should provide
+    the rough purpose for it.
+``category_id``
+    This is the *module category*, it serves to associate groups with a
+    business set of modules (essentially an application and all other modules
+    expanding on that application) and make them an exclusive selection, there
+    are also a few special-case categories for "extra" groups which don't fit
+    neatly in that scheme.
+``implied_ids``
+    Groups which are implied (automatically added) when giving this group to
+    a user e.g. a manager group might *imply* the corresponding user group, or
+    not depending on the security setup.
+``comment``
+    Additional notes on the purpose of the group, if the name alone might not
+    be clear enough, or if it warrants additional explanations.
+
+An other thing we want to do is to create a dedicated user with which we can
+test access rights, and whose group we can mess with
+
+.. exercise:: Create a user via the interface: as the administrator,
+
+    - open the Settings application then Invite a New User ``agent@example.org``
+    - click on the email under "Pending Invitations"
+    - edit the account and remove the Invoicing: Billing Administrator" group
+    - create a *private window* (or private / container tab, or a different
+      browser), and open the subscription link in that window, this way you
+      will be able to pick a password for your agent and have "them"
+      automatically logged in without being logged out as the administrator
+
+.. exercise::
+
+    Create your user and manager groups, then give the "manager" group to the
+    administrator and the "user" to the agent via the Odoo interface.
+
+    .. tip:: If you give your module a category (in its manifest), that will
+             automatically generate an xid of the form
+             :samp:`module_category_{name}` which you can use for the groups
+             (``name`` being the lower-cased version of the category name in the
+             manifest, with spaces replaced by underscores).
```

Comment:
> add an exercise to specify a module category (it could be a section in itself)

This is more of a sideshow here, maybe the category should go into the main training instead? It's probsably a good idea to properly setup the category anyway.

> use base.module_category_hidden with a basic explanation

That's very much *not* the intent, as that means the settings are checkboxes in the mess that is the hidden "technical" section of the ACL

![Screenshot from 2020-10-22 13-08-11](https://user-images.githubusercontent.com/7571158/96863937-ef112f80-1467-11eb-9729-d7829b4aabed.png)

Reply:
The thing is that there is no real use of the category in the core training. I think it makes more sense to introduce the concept when it needs to be connected with the security groups.